PyPortal_EZ_Make_Oven/code.py

```diff
@@ -5,19 +5,17 @@
 import gc
 import board
 import busio
+import audioio
 import audiocore
 import displayio
 import digitalio
-from adafruit_pyportal import PyPortal
 from adafruit_bitmap_font import bitmap_font
 from adafruit_display_text import bitmap_label as label
 from adafruit_display_shapes.circle import Circle
 from adafruit_button import Button
 import adafruit_touchscreen
 from adafruit_mcp9600 import MCP9600
 
-pyportal = PyPortal()
-
 TITLE = "EZ Make Oven Controller"
 VERSION = "1.3.0"
```

Comment:
We should update the version here.  _(I should have done this too, and didn't unfortunately)_ 

Since no functionality changed I think we can go with `"1.3.1"`

Reply:
Version bumped